src/WinXP/apps/DisplayProperties/ScreenSaverTab.js

```diff
@@ -35,8 +135,90 @@ const ScreenSaver = styled.div`
       background-color: #2f71cd;
     }
   }
-  .config {
-    margin: 3px 0px 0px 0px;
+
+  .config-area {
+    margin-top: 4.5px;
+  }
+
+  .settings {
+    position: relative;
+    border: 1px solid #e1e1d4;
+    border-radius: 5px;
+    font-size: 11px;
+    height: 80px;
+
+    & p {
+      margin-left: 82px;
+      margin-top: 7px;
+    }
+
+    & .power-button {
+      position: absolute;
+      right: 9px;
+      top: 45px;
+    }
+  }
+
+  .settings:nth-child(2) {
+    height: 87.5px;
+    margin-top: 4px;
+  }
+
+  legend {
+    margin-left: 9px;
+    color: #1a66c4;
+  }
+
+  .position-input {
+    border-radius: 0;
+    border-color: grey;
+    font-size: 12px;
+    margin-left: 9px;
+    margin-top: 6.5px;
+    width: 167px;
+    height: 22px;
+    &:focus {
+      color: #fff;
+      background-color: #2f71cd;
+      box-shadow: inset 0px 0px 0px 2px #fff;
+      outline: none;
+    }
+  }
+
+  .arrow-down {
+    position: absolute;
+    left: 159px;
+    top: 7px;
+    width: 16px;
+    height: 20px;
+    pointer-events: none;
+  }
+
+  .button-group {
+    display: inline-block;
+    position: absolute;
+    top: 5px;
+  }
+
+  .quickSettings {
```

Comment:
Why are some classes camelCase and others in kebab-case?

Reply:
My bad, fixed.